lib/ansible/modules/cloud/docker/docker_compose.py

```diff
@@ -16,65 +16,67 @@
 
 module: docker_compose
 
-short_description: Manage docker services and containers.
+short_description: Manage docker containers with Docker Compose.
 
 version_added: "2.1"
 
 author: "Chris Houseknecht (@chouseknecht)"
 
 description:
-  - Consumes docker compose to start, shutdown and scale services.
+  - Uses Docker Compose to start, shutdown and scale services.
   - Works with compose versions 1 and 2.
-  - Compose can be read from a docker-compose.yml (or .yaml) file or inline using the C(definition) option.
+  - Configuration can be read from a C(docker-compose.yml) or C(docker-compose.yaml) file or inline using the I(definition) option.
   - See the examples for more details.
   - Supports check mode.
-  - This module was called C(docker_service) before Ansible 2.8. The usage did not change.
+  - This module was called I(docker_service) before Ansible 2.8. The usage did not change.
```

Comment:
```suggestion
  - This module was called C(docker_service) before Ansible 2.8. The usage did not change.
```
This can stay `C(...)`. It's not an option name, and the same formatting (of the same sentence) is used in many other modules :)